build.sbt

```diff
@@ -1,21 +1,22 @@
 import sbtassembly.AssemblyPlugin.defaultUniversalScript
 
-ThisBuild / scalaVersion := "2.12.12"
+ThisBuild / scalaVersion := "2.13.6"
 ThisBuild / organization := "com.github.cross-language-cpp"
 
 lazy val djinni = (project in file("."))
   .configs(IntegrationTest)
   .settings(
     name := "djinni",
     Defaults.itSettings,
-    libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.5" % "it",
-    libraryDependencies += "org.scala-lang.modules" %% "scala-parser-combinators" % "1.1.2",
-    libraryDependencies += "org.yaml" % "snakeyaml" % "1.26",
+    libraryDependencies += "org.scalactic" %% "scalactic" % "3.2.10",
```

Comment:
@a4z do you remember why scalatic was added as a dependency in this PR? I cannot find any imports that make use of it. :/

Reply:
I plan(ned) to use it, but haven't come that far (so far) 
there are some nice features for testing in it